azure-communication-ui/chat/build.gradle

```diff
@@ -37,6 +37,10 @@ dependencies {
     implementation "androidx.fragment:fragment-ktx:$androidx_fragment_ktx_version"
     implementation "androidx.navigation:navigation-fragment-ktx:$androidx_navigation_fragment_ktx_version"
 
+
+    api ("com.azure.android:azure-communication-chat:1.1.0")
+    implementation 'com.azure.android:azure-communication-common:1.0.0'
```

Comment:
I think this will be api too, as contoso will use type too.

Reply:
Will clean up the gradle after https://github.com/Azure/communication-ui-library-android/pull/415 get merged.